draft-ietf-quic-transport.md

```diff
@@ -5018,10 +5034,16 @@ sequence number, or if a sequence number is used for different connection
 IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
+If the Retire Prior To field is present it is a request for the peer to
+immediately retire all connection IDs with a sequence number less than the
+specified value.  For more information see {{retiring-cids}}.  This field
+should only ever increase, though reordering and retransmission of older packets
+may make it seem to decrease.  So any decrease in this field should be ignored.
+
```

Comment:
Retransmission of packets is not the right term here but I agree with the overall intent. To be consistent with other monotonic fields perhaps says something like the SHOULD not send a smaller value unless a previous frame is retransmitted verbatim. MUST ignore frames with a prior to value less than one previously received.

This means that the entire frame is to be ignored, not just the prior to field.

Reply:
Or, I'm not sure: it is a burden for the receiver to check the prior to field. The sequence number ought to be the only value ruling validity. One could state that the receiver MAY issue a protocol violation if the a sequence number is received hat contradict monotonic increase in prior to values.

Reply:
We have the same problem with flow control values. We allow for implementations to naively retransmit packets that were lost, as is, so it's possible you have already updated the prior to field in a more recent packet, but then retransmit an older packet.

Reply:
I know, I just suggest a wording more close like those (don't recall the details)